frontend/src/modules/organization/components/edit/organization-edit-logo.vue

```diff
@@ -0,0 +1,133 @@
+              </div>
+            </template>
+          </lf-avatar>
+        </div>
+        <div class="flex-grow">
+          <lf-field label-text="Image URL">
+            <lf-input
+              v-model="form.logo"
+              :invalid="$v.logo.$invalid"
+              @blur="$v.logo.$touch()"
+              @update:model-value="$v.logo.$touch()"
+            >
+              <template #suffix>
+                <lf-icon
+                  v-if="form.logo"
+                  name="close-circle-line"
+                  :size="20"
+                  class="text-gray-300 cursor-pointer"
+                  @click="form.logo = ''"
+                />
+              </template>
+            </lf-input>
+            <lf-field-messages
+              :validation="$v.logo"
+              :error-messages="{
+                url: 'This URL address is not valid',
+              }"
+            />
+            <lf-field-message v-if="!$v.logo.$invalid" type="hint">
+              Choose a 1:1 aspect ratio (square) to better fit your image.
+            </lf-field-message>
+          </lf-field>
+        </div>
+      </div>
+      <div class="px-6 py-4 gap-4 flex items-center justify-end">
+        <lf-button type="secondary-ghost" @click="close">
+          Cancel
+        </lf-button>
+        <lf-button
+          type="primary"
+          :disabled="$v.$invalid || props.organization.logo === form.logo"
+          :loading="sending"
+
+          @click="update()"
+        >
+          Update logo
+        </lf-button>
+      </div>
+    </template>
+  </lf-modal>
+</template>
+
+<script setup lang="ts">
+import LfAvatar from '@/ui-kit/avatar/Avatar.vue';
+import LfModal from '@/ui-kit/modal/Modal.vue';
+import { computed, reactive, ref } from 'vue';
+import LfButton from '@/ui-kit/button/Button.vue';
+import { url } from '@vuelidate/validators';
+import useVuelidate from '@vuelidate/core';
+import LfInput from '@/ui-kit/input/Input.vue';
+import LfField from '@/ui-kit/field/Field.vue';
+import LfFieldMessage from '@/ui-kit/field-message/FieldMessage.vue';
+import LfFieldMessages from '@/ui-kit/field-messages/FieldMessages.vue';
+import Message from '@/shared/message/message';
+import LfIcon from '@/ui-kit/icon/Icon.vue';
+import { Organization } from '@/modules/organization/types/Organization';
+import { useOrganizationStore } from '@/modules/organization/store/pinia';
+
+const props = defineProps<{
+  modelValue: boolean,
+  organization: Organization,
+}>();
+
+const emit = defineEmits<{(e: 'update:modelValue', value: boolean): any}>();
+
+const { updateOrganization } = useOrganizationStore();
+
+const form = reactive({
+  logo: props.organization.logo,
+});
+
+const rules = {
+  logo: {
+    url,
+  },
+};
+
+const $v = useVuelidate(rules, form);
+
+const sending = ref<boolean>(false);
+const update = () => {
+  sending.value = true;
+  updateOrganization(props.organization.id, {
+    logo: form.logo,
+  })
```

Comment:
It seems that backend is throwing an error when trying to update the organization logo:
`Cannot read properties of undefined (reading 'filter')`

Is it something you can also look into?